File: app/src/main/java/org/tvheadend/tvhclient/ui/features/startup/SplashActivity.java
package org.tvheadend.tvhclient.ui.features.startup;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        Intent intent = new Intent(this, StartupActivity.class);
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/org/tvheadend/tvhclient/ui/features/startup/SplashActivity.kt
package org.tvheadend.tvhclient.ui.features.startup

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import org.tvheadend.tvhclient.ui.features.MainActivity

class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val intent = Intent(this, MainActivity::class.java)
        startActivity(intent)
        finish()
    }
}
